cmd/panacead/cmd/dep/hash_data.go

```diff
@@ -0,0 +1,41 @@
+package dep
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/cyberphone/json-canonicalization/go/src/webpki.org/jsoncanonicalizer"
+	"github.com/spf13/cobra"
+)
+
+func HashDataCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hash-data [data-file-path]",
+		Short: "Hash a data by sha256 algorithm",
+		Long: `
+This command can hash a data by sha256 algorithm. The data will be encoded to canonical JSON format and printed as a hash value.`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
func HashDataCmd() *cobra.Command {
	cmd := &cobra.Command{
		Use:   "hash-data [json-file-path]",
		Short: "Hash a JSON data by sha256 algorithm",
		Long: `
This command can hash a JSON data by sha256 algorithm. The data will be encoded to canonical JSON format and printed as a hash value.`,
		Args: cobra.ExactArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
```
It would be better to mention that this command targets only JSON data since it does JSON canonical encoding implicitly.

Reply:
Fixed it in [4f76219](https://github.com/medibloc/panacea-core/pull/654/commits/4f76219ab2f7c25d26b4c0be50ea73a77be22974).